pkg/proxy/snapshot.go

```diff
@@ -104,7 +104,27 @@ func (p *Proxy) SnapshotCloneStatus(ctx context.Context, req *rpc.ProxyEngineReq
 		Status: map[string]*eptypes.SnapshotCloneStatusResponse{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.